index.html

```diff
@@ -18,68 +21,122 @@
     <link rel='apple-touch-icon' sizes='144x144' href='/images/favicons/apple-icon-144x144.png'>
     <link rel='apple-touch-icon' sizes='152x152' href='/images/favicons/apple-icon-152x152.png'>
     <link rel='apple-touch-icon' sizes='180x180' href='/images/favicons/apple-icon-180x180.png'>
-    <link rel='icon' type='image/png', sizes='192x192', href='/images/favicons/android-icon-192x192.png'>
-    <link rel='icon' type='image/png', sizes='32x32', href='/images/favicons/favicon-32x32.png'>
-    <link rel='icon' type='image/png', sizes='96x96', href='/images/favicons/favicon-96x96.png'>
-    <link rel='icon' type='image/png', sizes='16x16', href='/images/favicons/favicon-16x16.png'>
-
+    <link rel='icon' type='image/png' , sizes='192x192' , href='/images/favicons/android-icon-192x192.png'>
+    <link rel='icon' type='image/png' , sizes='32x32' , href='/images/favicons/favicon-32x32.png'>
+    <link rel='icon' type='image/png' , sizes='96x96' , href='/images/favicons/favicon-96x96.png'>
+    <link rel='icon' type='image/png' , sizes='16x16' , href='/images/favicons/favicon-16x16.png'>
 </head>
+
 <body>
     <!--[if lt IE 7]>
-    <![endif]-->
-
-    
-        <nav class="navbar navbar-inverse navbar-fixed-top" role="navigation">
-            <!-- Collect the nav links, forms, and other content for toggling -->
-            <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
-                <ul class="nav navbar-nav">
-                    <li>
-                        <a href="../index.html">Home</a>
-                    </li>
-                    <li>
-                        <a href="../navigation.html">All lessons</a>
-                    </li>
-                </ul>
+        <![endif]-->
+    <nav class="navbar navbar-inverse navbar-fixed-top" role="navigation">
+        <!-- Collect the nav links, forms, and other content for toggling -->
+        <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+            <ul class="nav navbar-nav">
+                <li>
+                    <a href="../index.html">Home</a>
+                </li>
+                <li>
+                    <a href="../navigation.html">All lessons</a>
+                </li>
+            </ul>
+        </div>
+    </nav>
+
+    <div class="container-fluid">
+
+        <div class="fond">
+            <div id="forb" style="float:left;" class="style_prevu_kit"></div>
+            <div id="button" style="float:left;" class="button-flip">
+                <a id="button" class="active" href="#text">
```

Comment:
it is a no-op to repeat ids through the same page. Ids should be unique and they should identify the component. If you need to re-use style, you should move it to a class. Since the idea is for style to be as reusable as possible I would avoid using ids to style a component.